tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/notificator/DiscordNotificator.java

```diff
@@ -277,8 +292,8 @@ public void notifyBuildFailedToStart(@NotNull SRunningBuild sRunningBuild, @NotN
     @Override
     public void notifyLabelingFailed(@NotNull Build build, @NotNull VcsRoot vcsRoot, @NotNull Throwable throwable, @NotNull Set<SUser> users) {
         String title = "Labeling failed";
-        String description = "Labeling of build with the ID " + build.getBuildId() + " has failed!";
-        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + build.getBuildId();
+        String description = "Labeling of build with the ID " + build.getBuildNumber() + " has failed!";
+        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + build.getBuildNumber();
```

Comment:
Please continue to use `sRunningBuild.getBuildId();` here.
The URL expects the build id. If the build number is customized in the settings, the URL won't work anymore.